cmd/goci/main.go

```diff
@@ -47,6 +49,13 @@ func run(mode string) error {
 	}
 
 	switch mode {
+	case "validate":
+		err := validateRun()
+		if err != nil {
+			fmt.Fprintln(os.Stdout, err)
```

Comment:
Now that I see this, separating the validation error from other possible errors might be better. We would want to be able to tell the difference between a transient error vs a validation failure. Additionally, printing the message here then returning the error will cause it to be printed twice. It is also worth thinking through what kind of exit coded we should have for a validation error vs a regular error. Another option to handle this would be to create a validation error type, and discern the errors that way instead of separating them.